@ramonschepers - just gave your fork a shot ahead of official release (needed a sleep function :) ).  Anyway ran your example deepsleep.ino and received the following error:
```
C:\Users\CyberPalin\Documents\Arduino\libraries\TeensyThreads-master\TeensyThreads.cpp: In member function 'void Threads::idle()':

C:\Users\CyberPalin\Documents\Arduino\libraries\TeensyThreads-master\TeensyThreads.cpp:494:59: error: 'enter_sleep' was not declared in this scope
   volatile int time_spent_asleep = enter_sleep(substractor);
```
I also tried running blink.ino and received the same error.  

Running a T3.5 with TD1.4 and IDE 1.85 for reference. 

thanks mike

--- a/TeensyThreads.h
+++ b/TeensyThreads.h
@@ -156,6 +156,9 @@ class ThreadInfo {
 
 extern "C" void unused_isr(void);
 
+//added
+extern "C" int enter_sleep(int ms);
+
 typedef void (*ThreadFunction)(void*);
 typedef void (*ThreadFunctionInt)(int);
 typedef void (*ThreadFunctionNone)();

--- a/examples/Deepsleep/Deepsleep.ino
+++ b/examples/Deepsleep/Deepsleep.ino
@@ -12,7 +12,7 @@ const int LED = 13;
 const int UserLED = 2;
 
 //your sleeping funcion
-extern int enter_sleep(int ms) {
+int enter_sleep(int ms) {
   timer.setTimer(ms);//set sleep time in milliseconds  
   Snooze.hibernate( config ); //go to actual sleep
   //additional, one can use the RTC or a low power timer

